assignment-2/src/view/QuestionList.js

```diff
@@ -1,89 +1,69 @@
-import React from "react"
+import React from "react";
+import QuestionSearchBar from "./question/QuestionSearchBar";
+import QuestionButtons from "./question/QuestionButtons";
+import QuestionTile from "./question/QuestionTile";
+
+const QuestionList = ({
+    questions,
+    onSearchTitle,
+    newQuestion,
+    onChangeNewQuestionProperty,
+    onLogOut,
+    onClick,
+    onSearchTag,
+    tagOrTitle,
+    voteUp,
+    voteDown,
+    onSearchBarChange,
+    onClickNewQuestion
+}) => (
+    // <head>
```

Comment:
You should not include styles like that, that's why :P JSX should just include HTML and, in extreme scenarios, inline styles. You should include (S)CSS files with the `import` keyword.